src/cpio/client_providers/kms_client_provider/azure/azure_kms_client_provider.cc

```diff
@@ -215,24 +313,21 @@ void AzureKmsClientProvider::OnDecryptCallback(
 std::unique_ptr<KmsClientProviderInterface> KmsClientProviderFactory::Create(
     KmsClientOptions options,
     RoleCredentialsProviderInterface* role_credentials_provider,
-    AsyncExecutorInterface* io_async_executor) noexcept {
+    core::AsyncExecutorInterface* io_async_executor) noexcept {
   // We uses GlobalCpio::GetGlobalCpio()->GetHttpClient() to get http_client
   // object instead of adding it to KmsClientProviderFactory::Create() as a new
   // parameter. This is to prevent the existing GCP and AWS implementations from
   // being changed.
-  auto cpio_ = &GlobalCpio::GetGlobalCpio();
```

Comment:
This change appears inconsistent with the new interfaces in 3.4.0.

Reply:
this line was not changed from dev: auto cpio_ = &GlobalCpio::GetGlobalCpio();

What is the issue?